File: PersoMechant.py
from Personnage import Personnage

class PersoMechant(Personnage):


    def __init__(self, malice=None):
        super().__init__()
        self.malice = malice

    def __str__(self):
        return f"PersoMechant{super().__str__()} [Malice: {self.malice}]"

File: StarWars_Java/src/main/java/PersoMechant.java
package starwars;


/* 
@Author Yanis
@Projet : StarWars
*/

public class PersoMechant extends Personnage{
	private Boolean Coteobscure;
	
	public PersoMechant(Boolean Coteobscure) {
		super();
        this.Coteobscure = Coteobscure;
    }

    
	public Boolean getCoteobscure() {
		return Coteobscure;
	}
	public void setCoteobscure(Boolean coteobscure) {
		Coteobscure = coteobscure;
	}
	@Override
	public String toString() {
		return "PersoMechant"+super.toString()+" [Coteobscure=" + Coteobscure + "]";
	}


	
	
}